### DISCUSSION TITLE
JDBC Sinks Discussion

### DISCUSSION THREAD

**Discussion by flomickl** [apache/streampipes#1313] [Dev]
I think the last time this was discussed was during
https://github.com/apache/streampipes/issues/512

We should document the supported type and make it easier for new JDBC sinks like  #1236 
and validate how to implement other database types.

https://github.com/apache/streampipes/blob/dev/streampipes-extensions/streampipes-sinks-databases-jvm/src/main/java/org/apache/streampipes/sinks/databases/jvm/jdbcclient/model/DbDataTypes.java

and handled here https://github.com/apache/streampipes/blob/dev/streampipes-extensions/streampipes-sinks-databases-jvm/src/main/java/org/apache/streampipes/sinks/databases/jvm/jdbcclient/model/DbDataTypeFactory.java

First and Older Drafts can be found here:
https://cwiki.apache.org/confluence/x/Qg54CQ

We should also discuss what new Sinks can / should be implemented in the future

- [ ] Microsoft SQL
- [ ] POSTGIS
- [ ] Oracle DB
- [ ]  ...

Any other topics for JDBC I forgot?

**Comment by flomickl** [comment]
@bossenti is there no label for sinks?

**Reply by bossenti** [reply]
`pipeline-elements` is intended to be used for sinks as well :)
